### PR TITLE
Show the new media manager for browsers that do support ECMAScript 5

### PR DESCRIPTION
**Pull request by dneukirchen**
Pull Request for Issue # .

### CODE REVIEW

File: administrator/components/com_media/media.php [hunk 1]
```diff
@@ -15,32 +17,44 @@
...
-if (!$user->authorise('core.manage', 'com_media') && (!$asset or (!$user->authorise('core.edit', $asset)
-	&& !$user->authorise('core.create', $asset)
-	&& count($user->getAuthorisedCategories($asset, 'core.create')) == 0)
-	&& !($user->id == $author && $user->authorise('core.edit.own', $asset))))
+if (!$user->authorise('core.manage', 'com_media') && (!$asset || (!$user->authorise('core.edit', $asset)
+			&& !$user->authorise('core.create', $asset)
+			&& count($user->getAuthorisedCategories($asset, 'core.create')) == 0)
+		&& !($user->id == $author && $user->authorise('core.edit.own', $asset)))
+)
 {
```
Comment (laoneo) [inline administrator/components/com_media/media.php:24]: No need to more intend the code

File: administrator/components/com_media/media.php [hunk 2]
```diff
@@ -15,32 +17,44 @@
...
 
-$params = JComponentHelper::getParams('com_media');
+// Show the new media manager for browsers that do support ECMAScript 5
+$client   = JFactory::getApplication()->client;
+$basePath = JPATH_COMPONENT_ADMINISTRATOR;
 
-// Load the helper class
-JLoader::register('MediaHelper', JPATH_ADMINISTRATOR . '/components/com_media/helpers/media.php');
+if ($client->browser == WebClient::IE && $client->version <= 8)
+{
```
Comment (laoneo) [inline administrator/components/com_media/media.php:33]: For clarification, I would add here a comment, why we do something special for IE 8 and below

Comment (dneukirchen) [reply administrator/components/com_media/media.php:33]: done

File: administrator/components/com_media/media.php (hunk 1 above)
Comment (dneukirchen) [reply administrator/components/com_media/media.php:24]: done

**Pull request merged**